### PR TITLE
Add 2022 Elixir confs (preliminary)

### PR DESCRIPTION
**Pull request by Tuxified**
Hi,

Since there were no Elixir conferences for the upcoming year on this site, I decided to try to contribute include them. Only problem is these conferences are hybrid (both online and in person), so I took the liberty of adding another boolean prop: "in-person" so confs can be both. 

Let me know what you think :D

### CODE REVIEW

**Comment by nimzco** [comment]
Hey @Tuxified ! Thank you so much for your contribution!

We actually do have the notion of hybrid conferences. You just have to set "online" to true and add the location 🙃

**Comment by Tuxified** [comment]
Hi @nimzco, thanks for the quick reply. I made the wrong assumption based on a previous conf which was wrongfully flagged as online only, so the superfluous "in-person" flag is removed now. Thanks for letting me know :D

**Comment by nimzco** [comment]
Absolutely no worries! Thanks again for your contribution! Let's get these merged in! 🎉

File: conferences/2022/elixir.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    "twitter": "@CodeSyncGlobal",
+    "cocUrl": "https://codesync.global/about-us/#Conduct"
+  },
+  {
+    "name": "ElixirConf EU",
+    "url": "https://www.elixirconf.eu",
+    "startDate": "2022-04-07",
+    "endDate": "2022-04-07",
+    "city": "London",
+    "country": "UK",
```
Comment (nimzco) [inline conferences/2022/elixir.json:30]:
```suggestion
    "country": "U.K.",
```

File: conferences/2022/elixir.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
+[
+  {
+    "name": "Lambda Days",
+    "url": "https://www.lambdadays.org/lambdadays2022/",
+    "startDate": "2022-02-10",
+    "endDate": "2022-02-11",
+    "online": true,
+    "city": "Krakow",
+    "country": "Poland",
+    "twitter": "@LambdaDays"
```
Comment (nimzco) [inline conferences/2022/elixir.json:10]:
```suggestion
    "twitter": "@LambdaDays",
```

File: conferences/2022/elixir.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    "cocUrl": "https://www.lambdadays.org/lambdadays2022/#code-of-conduct"
+  },
+  {
+    "name": "Code BEAM Europe",
+    "url": "https://codesync.global/conferences/code-beam-sto-2022/",
+    "startDate": "2022-05-19",
+    "endDate": "2022-05-20",
+    "online": true,
+    "city": "Stockholm",
+    "country": "Sweden"
```
Comment (nimzco) [inline conferences/2022/elixir.json:20]:
```suggestion
    "country": "Sweden",
```

File: conferences/2022/elixir.json [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    "url": "https://www.elixirconf.eu",
+    "startDate": "2022-04-07",
+    "endDate": "2022-04-07",
+    "city": "London",
+    "country": "U.K.",
+    "online": true,
+    "twitter": "@elixirconfeu",
+    "cocUrl": "https://www.elixirconf.eu/assets/images/CoCEU.pdf",
+    "cfpUrl": "https://eventil.com/events/elixirconf-eu-2022/cfp",
+    "cfpEndDate": "2021-12-09",
```
Comment (nimzco) [inline conferences/2022/elixir.json:35]:
```suggestion
    "cfpEndDate": "2021-12-09"
```

**Pull request merged**